README.md

```diff
@@ -1,6 +1,6 @@
 # README 
 [![Maintainability](https://api.codeclimate.com/v1/badges/ce8783c839063c27b944/maintainability)](https://codeclimate.com/github/Fermented-Software/Kimchi/maintainability)
-[![Coverage Status](https://coveralls.io/repos/github/egydiopacheco/Kimchi/badge.svg?branch=main)](https://coveralls.io/github/egydiopacheco/Kimchi?branch=main)
+[![Coverage Status](https://coveralls.io/repos/github/Fermented-Software/Kimchi/badge.svg?branch=main)](https://coveralls.io/github/Fermented-Software/Kimchi?branch=main)
 ![GA Release Workflow main](https://github.com/egydiopacheco/Kimchi/actions/workflows/ci-release.yaml/badge.svg?branch=main)
 ![GA Pull Request Workflow main](https://github.com/egydiopacheco/Kimchi/actions/workflows/pr-test.yaml/badge.svg?branch=main)
```

Comment:
Do we still keep the line 5 (Pull Request Workflow badge) even with the change of the pr-test.yaml file?

Reply:
Good one! I think it should work if we swap the name of the file

Reply:
I think we keep the badge but make de required alterations so it reflects what is in the code, like it's name.

Reply:
Changed it on d16843f7f683fb3c9e23b1415e278a57a30eff4d